core/src/bin/publishRelease.sh

```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-SBT_OPTS=-Dproject.isSnapshot=false sbt +publish
+sbt -J-Dproject.isSnapshot=false +publish
```

Comment:
`project.isSnapshot` was not getting respected by the following code, causing the jfrog artifact to always have "SNAP" at the end:

https://github.com/broadinstitute/rawls/blob/6ac29cea826d39d6e72e1aaf0dbbfadce3c168ca/project/Version.scala#L18

I have no idea when this broke.